crates/librqbit/src/torrent_state/live/mod.rs

```diff
@@ -1018,17 +1020,27 @@ impl PeerHandler {
 
         pe.value_mut().state.set(PeerState::Dead, &pstats);
 
+        if self.incoming {
+            // do not retry incoming peers
+            return Ok(());
+        }
+
         let backoff = pe.value_mut().stats.backoff.next_backoff();
 
         // Prevent deadlocks.
         drop(pe);
 
         if let Some(dur) = backoff {
+            debug!(
```

Comment:
please remove this message entirely, it's just noise.

The error_span!() below does it all already, and it's printed multiple times as part of spawn().